docs/install.md

```diff
@@ -140,8 +142,19 @@ use to protect your routes, `session`, `tokens`, and `chained`. The first two co
 to see if the user is logged in through either of authenticators, allowing a single API endpoint to
 work for both an SPA using session auth, and a mobile app using access tokens. The fourth, `auth-rates`,
 provides a good basis for rate limiting of auth-related routes.
+These can be used in any of the [normal filter config settings](https://codeigniter4.github.io/CodeIgniter4/incoming/filters.html#globals), or [within the routes file](https://codeigniter.com/user_guide/incoming/routing.html#applying-filters).
```

Comment:
Not sure if you meant to link to two different versions of the UG?

Reply:
Sorry to be unclear! codeigniter4.github.io is the latest dev build of the UG and codeigniter.com is the public stable one. You used both versions- I wasn't sure if there was something in Filters on the dev version you wanted, or if it was simply a copy-paste mistake.

Reply:
`https://codeigniter.com/user_guide/` is the official one for the stable release, and it is used in the existing docs.
Probably we should use it.

`https://codeigniter4.github.io/CodeIgniter4/` is the current UG of `develop` branch.
`https://codeigniter4.github.io/userguide/` is the same as `https://codeigniter.com/user_guide/`, but hosted by GitHub Pages.

Reply:
Oh! @MGatner , as you said, it was a copy-paste mistake.
Thank you Kenjis.

**Done.**